README.md

```diff
@@ -1822,6 +1822,22 @@ get do
 end
 ```
 
+The above example may have been requested as follows:
```

Comment:
Let's add a section called `Header Case Handling`? This way it appears in the TOC and can be linked.

Reply:
Adding the section heading awkwardly makes it run into the next section on the response headers and I don't want to imply that this handling applies to responses.  So I'll add a Request and Response section title as well to be clear.  I hope this is ok.

Reply:
👍